docs/emacs.org

```diff
@@ -2,34 +2,124 @@
 
   [[./edit-with-emacs-demo.gif/]]
 
-  Any text, in just any app can be edited using Emacs. The idea is simple - you press a dedicated key-combo (default: =Cmd+Ctrl+O=), Spacehammer copies the existing text & calls ~emacsclient~, then it invokes a function that opens a buffer and pastes the text into it. After you're done editing the text in Emacs, you press =C-c C-c=, it grabs the text, switches back to the program & pastes the text back in there. It works surprisingly well.
+  Any text, in just any app can be edited using Emacs. The idea is simple - you press a dedicated key-combo (default: =Cmd+Ctrl+O=), Spacehammer copies the existing text & calls ~emacsclient~, which invokes a function that opens a buffer and pastes the text into it. Once you finish editing, you press =C-c C-c=, Emacs grabs the text, switches back to where you were before and pastes the new text in there. It works surprisingly well.
```

Comment:
But: what do you mean by "just" any app. You mean "just about any app" or "any app". "Just any app" sounds strange.